### ISSUE TITLE
keep port config after sleep

### ISSUE THREAD

**Issue by mamama1** [ee-quipment/ZeroPowerManager#4]
Hi,

is my assumption correct that as zpmPortDisableDigital() completely resets the port config of all pins to the power up state, one needs to either save the states first and reconfigure the ports after every wakeup?

also, what about an external button to wake up the MCU, would I need to attach the interrupt after disabling all pins so the button pin gets reconfigured as input? would it be advisable to use a hardware pullup resistor instead of the internal pullup?

basically I'm trying to build a device which deep sleeps with as lowest sleep current as possible but can be waked up by a button press or the internal RTC.

thanks!
